frontend/src/redux/drawerModeSlice.ts

```diff
@@ -0,0 +1,31 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { KubeObject } from '../lib/k8s/KubeObject';
+
+interface DrawerModeState {
+  isDetailDrawerEnabled: boolean;
+  selectedResource: KubeObject | undefined;
+}
+
+const getLocalDrawerStatus = (key: string) => localStorage.getItem(key) === 'true';
+
+const initialState: DrawerModeState = {
+  isDetailDrawerEnabled: getLocalDrawerStatus('detailDrawerEnabled'),
```

Comment:
'detailDrawerEnabled' is repeated, you should put it in a constant

Reply:
done